packages/components-native/src/Text/Text.test.tsx

```diff
@@ -219,3 +229,14 @@ describe("TypographyGestureDetector", () => {
     expect(textElement.props.collapsable).toBeUndefined();
   });
 });
+
+it("works when nested inside a native RN Text wrapper", () => {
```

Comment:
thinking about this more, let's remove the test. it's not something we want to support or people to do. 


plus if this failed it wouldn't tell us much about us. it would most likely be RNText failing, which isn't our responsibility.

I can see some instances in the app of people doing this and making links using this method, but I probably wouldn't encourage that over some of our more standardized ways of presenting and creating interactive text in the mobile space